src/data/set/intervals/order_iso.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro, Patrick Massot, Yury Kudryashov, Rémy Degenne
+-/
+import data.set.intervals.basic
+import order.hom.basic
+
+/-!
+# Lemmas about images of intervals under order isomorphisms.
```

Comment:
Why are you considering `order_iso` to be a big import?

Reply:
I agree this one may be a mistake. Happy to revert later if there's some reason too.